Publish-Devices/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Publish Different Devices</title>
+    
+    <link rel="stylesheet" href="css/app.css">
+    <script src="https://static.opentok.com/v2.15.0-beta/js/opentok.min.js"></script>
```

Comment:
Since 2.15 is out, should we change this?

Reply:
yes we should :) I'll update it now.